middleware/create-user-token.middleware.js

```diff
@@ -0,0 +1,32 @@
+const jwt = require('jsonwebtoken');
+
+const defaultPermissions = [
+  // read
+	'READ_ADDRESSES',
+	'READ_BATCHES',
+	'READ_DELIVERY_DATES',
+	'READ_DELIVERY_ZONES',
+	'READ_FORECAST',
+	'READ_FULFILLMENT_PLANS',
+	'READ_PACKING_ASSIGNMENTS',
+	'READ_PACKING',
+	'READ_SHIPMENTS',
+	'READ_SHIPPING_METHODS',
+	'READ_SHIPPING_RULES',
+	// write
+	'WRITE_ADDRESSES',
+	'WRITE_BATCHES',
+	'WRITE_DELIVERY_ZONES',
+	'WRITE_EASYPOST_WEBHOOK',
+	'WRITE_FORECAST',
+	'WRITE_JOBS',
+	'WRITE_PACKING',
+	'WRITE_PACKING_ASSIGNMENTS',
+	'WRITE_SHIPMENTS',
+	'WRITE_SHIPPING_METHODS',
+	'WRITE_SHIPPING_RULES',
+];
+
+module.exports = (permissions) => {
+	return jwt.sign({ permissions: [...defaultPermissions, ...permissions] }, 'abc123')
```

Comment:
I wouldn't merge on any defaults.  If you do that we won't be able to test the absence of a permission. Plus we'll be maintaining a bunch of permissions for all the apps, instead of the app itself knowing what permissions it cares about.